config-linux.md

```diff
@@ -564,7 +565,8 @@ Its value is either slave, private, or shared.
 
 ## Readonly Paths
 
-`readonlyPaths` will set the provided paths as readonly inside the container.
+**`readonlyPaths`** (array of strings, OPTIONAL) will set the provided paths as readonly inside the container.
+The values MUST be absolute paths.
```

Comment:
Maybe “…MUST be absolute paths in the [container mount namespace](glossary.md#container-namespace).”